lib/tasks/hierarchy_spike.rake

```diff
@@ -0,0 +1,48 @@
+require 'yaml'
+require "gds_api/publishing_api_v2"
+
+# The YAML data files are generated using a script
+# https://gist.github.com/tadast/defd1b6fc1e2eccd77f3
+
+GovukContentSchemaTestHelpers.configure do |config|
+  config.schema_type = 'publisher'
+  config.project_root = Rails.root
+end
+
+namespace :hierarchy_spike do
+  desc "Store hard-coded Service Manual Section hierarchy to the publishing API as a draft"
+  task publish_guides: [:environment] do
+    publish_from_file("guides.yml")
+  end
+
+  desc "Publish hard-coded Service Manual Section hierarchy that was previously saved as a draft"
+  task publish_sections: [:environment] do
+    publish_from_file("section_hierarchy.yml")
+  end
+
+  def publish_from_file(file_name)
+    publishing_api = GdsApi::PublishingApiV2.new(Plek.new.find('publishing-api'))
+    with_payload(file_name) do |payload|
+      validate_against_schema(payload)
+      publishing_api.put_content(payload[:content_id], payload)
+      publishing_api.put_links(payload[:content_id], payload) if payload[:links].present?
```

Comment:
I just learned from @elliotcm that putting links now takes immediate effect in production, therefore this leaves potential for data in `links` and `link_groups -> linked_items` to get out of sync as this operation is not atomic. Even worse, once we

Moreover, once we have a [UI for building section pages](https://github.com/alphagov/service-manual-publisher/pull/40#issuecomment-154373882) we'd need to work around this by making sure we always update the links hash only and every time time we publish content. @KushalP any thoughts?